tests/internal/e2e/test/specs_config.ts

```diff
@@ -1,62 +1,187 @@
+        "./**/**/vcast.rest.test.ts",
+        "./**/**/vcast.rest_2.test.ts",
+        "./**/**/vcast.rest_3.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
+    build_env_failure: {
+      specs: [
+        "./**/**/vcast_build_env_failure.test.ts",
+        "./**/**/vcast_build_env_after_failure.test.ts",
+      ],
+      env: {
+        VECTORCAST_DIR_TEST_DUPLICATE: process.env.VECTORCAST_DIR,
+        VECTORCAST_DIR: "",
+        VECTORCAST_ATG_DIR: "",
+      },
+      params: {
+        vcReleaseOnPath: false,
+      },
+    },
+    bugs: {
+      specs: [
+        "./**/**/vcast_testgen_bugs.test.ts",
+        "./**/**/vcast_testgen_bugs_2.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
+    flask_icon: {
+      specs: ["./**/**/vcast_testgen_flask_icon.test.ts"],
+      env: {},
+      params: {},
+    },
+    func_basis: {
+      specs: [
+        "./**/**/vcast_testgen_func_basis.test.ts",
+        "./**/**/vcast_testdel_func_basis.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
+    unit_basis: {
+      specs: [
+        "./**/**/vcast_testgen_unit_basis.test.ts",
+        "./**/**/vcast_testdel_unit_basis.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
+    env_basis: {
+      specs: [
+        "./**/**/vcast_testgen_env_basis.test.ts",
+        "./**/**/vcast_testdel_env_basis.test.ts",
+      ],
+      env: {},
+      params: {},
+    },
   };
 
-  if (vcast24) {
-    specGroups["func_atg"] = [
-      "./**/**/vcast_testgen_func_atg.test.ts",
-      "./**/**/vcast_testdel_func_atg.test.ts",
-    ];
+  if (useVcast24) {
+    specGroups["func_atg"] = {
+      specs: [
+        "./**/**/vcast_testgen_func_atg.test.ts",
+        "./**/**/vcast_testdel_func_atg.test.ts",
+      ],
+      env: {},
+      params: {},
+    };
 
-    specGroups["unit_atg"] = [
-      "./**/**/vcast_testgen_unit_atg.test.ts",
-      "./**/**/vcast_testdel_unit_atg.test.ts",
-    ];
+    specGroups["unit_atg"] = {
+      specs: [
+        "./**/**/vcast_testgen_unit_atg.test.ts",
+        "./**/**/vcast_testdel_unit_atg.test.ts",
+      ],
+      env: {},
+      params: {},
+    };
 
-    specGroups["env_atg"] = [
-      "./**/**/vcast_testgen_env_atg.test.ts",
-      "./**/**/vcast_testdel_env_atg.test.ts",
-    ];
+    specGroups["env_atg"] = {
+      specs: [
+        "./**/**/vcast_testgen_env_atg.test.ts",
+        "./**/**/vcast_testdel_env_atg.test.ts",
+      ],
+      env: {},
+      params: {},
+    };
 
-    specGroups["coded_tests"] = ["./**/**/vcast_coded_tests.test.ts"];
+    specGroups["coded_tests"] = {
+      specs: ["./**/**/vcast_coded_tests.test.ts"],
+      env: {},
+      params: {},
+    };
   }
 
   return specGroups;
 }
 
+export function getSpecsWithEnv(useVcast24: boolean) {
+  const specGroups = getSpecGroups(useVcast24);
+
+  Object.keys(specGroups).forEach((group) => {
+    const groupObj = specGroups[group];
+
+    // In that case we don t want the release path
```

Comment:
we don't want the release to be on path

Reply:
Changed comment in 3c5f99218af5dc6b962549004871a5fe6788d65a.